tests/unit/__init__.py

```diff
@@ -0,0 +1,27 @@
+test_bundle_spec = {
```

Comment:
I think this is fine, but also that storing this in a file in `matrix/tests/res/` would be cleaner. Fine either way